### ISSUE TITLE
How to choose the parameters in the function of lr_finder.find

### ISSUE THREAD

**Issue by xuzhang5788** [surmenok/keras_lr_finder#3]
How to set the values of start_lr, end_lr,  batch_size and epochs? In my model, I couldn't get the figures like your example. 

I used
lr_finder.find(X_train, y_train, start_lr=0.0001, end_lr=1, batch_size=512, epochs=5)
lr_finder.plot_loss(n_skip_beginning=20, n_skip_end=5)

Then I got a figure like this.
![loss_lr](https://user-images.githubusercontent.com/36318415/39496068-d38238fc-4d51-11e8-8fb5-85ef3c4a0769.png)

lr_finder.plot_loss_change(sma=20, n_skip_beginning=20, n_skip_end=5, y_lim=(-0.1, 0.1))
![image](https://user-images.githubusercontent.com/36318415/39496202-4f3af6d2-4d52-11e8-8853-ee3a8c9e5f5b.png)

 
Could you give me some advice to use other parameters to find a good lr? Many thanks

**Comment by surmenok** [comment]
One possibility is that start_lr is too high. Usually, for CNNs and small dense networks the parameters you use work fine, but for some cases (e.g. RNNs) LR 0.0001 could be too high. Try lower values, e.g. 0.000001. 
If it doesn't work even with low starting learning rate, it's possible that there are errors in the implementation of the model. 
Try to reduce the dataset to a small size (e.g. 100 examples) and check if the model can learn to predict correct labels on the training set (overfit to the training set).

**Comment by xuzhang5788** [comment]
Thank you so much. I will try it.
